### ISSUE TITLE
Shorts makes YT crash

### ISSUE THREAD

**Issue by bachig26** [Balackburn/YTLitePlus#46]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I read the FAQ

### Device info

- iOS/iPadOS version: 15.7.9
- Device model: iPhone SE 1st Gen
- Sideload tool (AltStore, Sideloadly, TrollStore,...): AltStore
- The specific version of YTLitePlus (**latest** or **newest** is **NOT** a version number!): v18.41.2-2.3.1

### Describe the bug

As you also know the existing issue with YTLite with Shorts [also mentioned in release: Shorts are not hidden everywhere (problem with YTLite tweak)].

If you click a Shorts clip in the Homepage to open, then the YouTube just crashes.

**Edit: Just found that even when you click on the Shorts tab in the Navigation bar makes the YT crash, it's everywhere where the Shorts available.**

### Steps to reproduce the issue

1. Open (YouTube) YTLitePlus
2. Click a Shorts in Homepage/ click the Shorts tab in Navigation bar
3. App crashes

### Are you using the newest version of YTLitePlus? If not, why?

✅ Yes, I'm using the latest version of YTLitePlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, It doesn't

### Additional context

@Balackburn ps. I wanted to let you know about this crash the happens from a existing known issue.

**Comment by Balackburn** [comment]
Hello I don't have this issue myself, could you try to disable every options in Settings>YTLite>Shorts ?

**Comment by bachig26** [comment]
> Hello I don't have this issue myself, could you try to disable every options in Settings>YTLite>Shorts ?

no, it still crashes for me even after disabling every options for shorts. 😢

**Comment by amitsingh-007** [comment]
Happening with me also on [v18.41.2-2.3.1 - YTLitePlus](https://github.com/Balackburn/YTLitePlus/releases/tag/v18.41.2-2.3.1) on iphone 14 pro max

**Comment by Balackburn** [comment]
> Happening with me also on [v18.41.2-2.3.1 - YTLitePlus](https://github.com/Balackburn/YTLitePlus/releases/tag/v18.41.2-2.3.1) on iphone 14 pro max

Does it only happens with latest version (18.41.2-2.3.1) I can't reproduce the issue unfortunately, please use previous versions in the meantime.

**Comment by bachig26** [comment]
> Does it only happens with latest version (18.41.2-2.3.1) I can't reproduce the issue unfortunately, please use previous versions in the meantime.

as for as I know, it's only reproducable in v18.41.2-2.3.1.

**Comment by amitsingh-007** [comment]
Can confirm, Shorts doesn't crash works if downgraded to [v18.40.1-2.3.1 - YTLitePlus](https://github.com/Balackburn/YTLitePlus/releases/tag/v18.40.1-2.3.1)

**Comment by Jad-Saleh** [comment]
> Can confirm, Shorts doesn't crash works if downgraded to [v18.40.1-2.3.1 - YTLitePlus](https://github.com/Balackburn/YTLitePlus/releases/tag/v18.40.1-2.3.1)

I’m getting this on iPhone 14 pro max with both the latest version and after downgrading to v18.40.1-2.3.1

it doesn’t happen every time I open a short but it’s about 80-90% of the time and if I scroll past that it’s almost guaranteed to crash. Let me know if there’s any way I can help debug

**Comment by baz-za** [comment]
> Hello I don't have this issue myself, could you try to disable every options in Settings>YTLite>Shorts ? 

This fixes it. but i need some options enabled in there so i just downgraded to 18.40

**Comment by Balackburn** [comment]
After checking, the following YTLite options do not work with the latest version of YT: 

1. Options>YTLite>Shorts>Hide Superthanks button
2. Options>YTLite>Shorts>Hide Description

Please disable them for now, tell me if it resolve the crash.

**Comment by Jad-Saleh** [comment]
> After checking, the following YTLite options do not work with the latest version of YT:
> 
> 1. Options>YTLite>Shorts>Hide Superthanks button
> 2. Options>YTLite>Shorts>Hide Description
> 
> Please disable them for now, tell me if it resolve the crash.

Unfortunately I’m getting the crash and all settings in YTLite>Shorts were already disabled

**Comment by Jad-Saleh** [comment]
> After checking, the following YTLite options do not work with the latest version of YT:
> 
> 1. Options>YTLite>Shorts>Hide Superthanks button
> 2. Options>YTLite>Shorts>Hide Description
> 
> Please disable them for now, tell me if it resolve the crash.

I think I solved it! My issue was fixed by switching from OLED dark mode in YTLitePlus settings to default theme.

**Comment by Alex-Garay** [comment]
> After checking, the following YTLite options do not work with the latest version of YT:
> 
>     1. Options>YTLite>Shorts>Hide Superthanks button
> 
>     2. Options>YTLite>Shorts>Hide Description
> 
> 
> Please disable them for now, tell me if it resolve the crash.

Turning off "Hide Description" seemed to fix it for me, thanks!

**Comment by RollinggThunder** [comment]
Its still there for me,crash when clicked on shorts

**Comment by Balackburn** [comment]
> Its still there for me,crash when clicked on shorts 

Could you send me your settings?

**Comment by RealLittlemrh** [comment]
Issue occurs for me when I turn off description and or superthanks on version 18.41.2-2.3.1

**Comment by Balackburn** [comment]
I think this is just a sign that you should stop watch them.

**Comment by Balackburn** [comment]
@RealLittlemrh @bachig26 Do you still have this issue with latest version?

**Comment by bachig26** [comment]
> @RealLittlemrh @bachig26 Do you still have this issue with latest version?

yes, this still exist for me.

for example,
when this option is toggle on `Options>YTLite>Shorts>Hide Description`, it crashes.

also as mentioned above by others, this option `Options>YTLite>Shorts>Hide Superthanks button` doesn't break the app for with this latest version.

**Comment by huandney** [comment]
The problem appears only with the oled dark theme.

**Comment by Balackburn** [comment]
@andyjhall @RealLittlemrh @bachig26 @huandney Do you still have this issue with latest version?

**Comment by andyjhall** [comment]
Given this a quick try and it appears to be working for me.

**Comment by Balackburn** [comment]
I close the issue, if this happens again let me notice and I will reopen it

**Comment by DivinorWieldor** [comment]
@Balackburn I'm experiencing the same problems. I have just downloaded the app so everything is latest and (almost) all settings are default.

I've tried to follow through the suggested solutions here. I have tried changing the theme through `YTLitePlus->Theme Options`. I have also gone over `Youtube Plus->Interface` but the OLED theme is already disabled. All options listed under `Youtube Plus->Shorts` are disabled. 

The video preview plays without any issues. When I click on a short (or go to the Shorts tab) the app generally crashes after a second. Rarely, the video may start playing when clicked but the app will still crash after a second. My current thinking is that the short content isn't the issue but some UI element is attempting to load but causes a crash somehow?

Update following some testing: it started working as soon as I disabled return YouTube Dislike. All other settings work fine and do not seem to cause a crash. I have tried to enable return dislike with alternative settings but I wasn't able to get it to not crash.

I'm using an iPhone 16 with iOS 18.5.
